src/apps/weblib/typeschema-api/config.yaml

```diff
@@ -109,14 +115,18 @@ definitions:
         type: object
         properties:
             x:
-                $ref: Channel
+                allOf:
```

Comment:
no need for `allOf` if `AxisChannel` inherits `Channel` properties

Reply:
https://github.com/vizzuhq/vizzu-lib/pull/632/commits/bbbc5238e656e889b13c997566ff6cb833ae195d